integration-tests/deps.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -e
+cd "$(dirname "${BASH_SOURCE[0]}")"
+
+PYTHON_VENV_DIR=${PYTHON_VENV_DIR:-"./bot/.venv"}
+
+pushd bot;
+if [ ! -d $PYTHON_VENV_DIR ];
+then
+    python3 -mvenv --without-pip $PYTHON_VENV_DIR
+    source $PYTHON_VENV_DIR/bin/activate
+    curl https://bootstrap.pypa.io/get-pip.py | python
+else
+    source $PYTHON_VENV_DIR/bin/activate
+fi
+pip3 install -e .
+pip3 install supervisor
+popd
+
+pushd client-rpc; npm install; popd
```

Comment:
Out of curiosity, `pushd` and `popd` is to ensure even `npm install` change the directory, it can still go back to the original one?

Reply:
> Out of curiosity, `pushd` and `popd` is to ensure even `npm install` change the directory, it can still go back to the original one?

Yeah, I think so, like a stack of absolute paths.